doc/source/dotnet.rst

```diff
@@ -42,14 +42,9 @@ application.
 
 Before interacting with any of the objects or APIs provided by the
 ``Python.Runtime`` namespace, calling code must have acquired the Python
-global interpreter lock by calling the ``PythonEngine.AcquireLock``
-method. The only exception to this rule is the
-``PythonEngine.Initialize`` method, which may be called at startup
-without having acquired the GIL.
-
-When finished using Python APIs, managed code must call a corresponding
-``PythonEngine.ReleaseLock`` to release the GIL and allow other threads
-to use Python.
```

Comment:
IMHO, when finished section needs corresponding replacement (maybe merged with the acquisition part.

Reply:
Better?